_posts/2019-03-13-there-is-no-such-a-thing-as-cross-services-composition.md

```diff
@@ -3,37 +3,37 @@ typora-copy-images-to: ..\img\posts\there-is-no-such-a-thing-as-cross-services-c
 typora-root-url: ..
 layout: post
 header_image: /img/posts/there-is-no-such-a-thing-as-cross-services-composition/header.jpg
-title: "There is no such a thing as cross Services ViewModel Composition"
```

Comment:
Switching from `services` to `service` might change the URL so you may need to create an alias. That said, it's not a big deal to leave it as `services`, but I would still switch it to lowercase.